template/web/wp-content/mu-plugins/studiometa-plugin-disabler/README.md

```diff
@@ -0,0 +1,10 @@
+# Studio Meta plugin disabler
+
+Disable plugins depends on environment.
+
+## Usage
+
+The following server environment variables must be set:
+```bash
+DISABLE_PLUGINS_[ENVIRONMENT]=example/example1.php
```

Comment:
**suggestion:** Only use the plugin's folder name here

Unless there is a specific interest in having both the folder and the main PHP file name?

Reply:
Plugins aren't always based on the `plugin_name/plugin_name.php` naming rule. Sometimes, they have odd naming structure.